api/PclusterApiHandler.py

```diff
@@ -551,9 +551,16 @@ def get_instance_types():
     return {"instance_types": sorted(instance_types, key=lambda x: x["InstanceType"])}
 
 
-def _get_user_roles(decoded):
-    return decoded[USER_ROLES_CLAIM] if USER_ROLES_CLAIM in decoded else ["user"]
+def _populate_identity(identity, decoded, claims):
```

Comment:
suggestion(non-blocking): for future contributions, I'd go with pure functions and immutability rather than modifying existing objects